frontend/components/form/checkbox/FormCheckboxBtn.vue

```diff
@@ -7,6 +7,7 @@
       [customColorClass]: modelValue && customColor,
       'bg-light-header text-light-special-text dark:text-dark-special-text dark:bg-dark-header':
```

Comment:
The styles can just be put in this one here as they have the same conditional of `!modelValue` :)